bip-9999.mediawiki

```diff
@@ -0,0 +1,151 @@
+
+===Specification===
+
+'''Redefinition of the Unit:'''
+
+* Internally, the smallest indivisible unit remains unchanged.
+* Historically, 1 bitcoin = 100,000,000 base units. Under this proposal, "1 bitcoin" equals that smallest unit.
+* What was previously referred to as "1 bitcoin" now corresponds to 100 million bitcoins under the new definition.
+
+'''Terminology:'''
+
+* The informal terms "satoshi" or "sat" are deprecated.
+* All references, interfaces, and documentation SHOULD refer to the base integer unit simply as "bitcoin."
+* The currency code "BTC" is unaffected by these changes, and continues to mean 100,000,000 base units.
+
+'''Display and Formatting:'''
+
+* Applications SHOULD allow users to toggle between the legacy BTC format (1 BTC = 100,000,000 base units) and the new integral format (1 bitcoin = 1 base unit).
+* Use of the ₿ symbol MAY be used to represent base-unit bitcoins but is OPTIONAL.
+
+Example 1:
+
+  * Old display: <code>0.00010000 Bitcoin<code>
+  * New display: <code>₿10,000<code> or <code>10,000 bitcoins<code> or <code>0.00010000 BTC<code>
+
+Example 2:
+
+  * Old display: <code>10.23486 Bitcoin<code>
+  * New display: <code>₿1,023,486,000<code> or <code>1,023,486,000 bitcoins<code> or <code>10.23486 BTC<code>
+
+Example 3:
+
+  * Old display: <code>0.345 BTC<code>
+  * New display: No changes required or <code>₿34,500,000<code> or <code>34,500,000 bitcoins<code>
+
+NOTE: Traditional number display abbreviations, like <code>2.5M<code> for millions, are also optional.
+
+'''Conversion:'''
+
+* Ledger and consensus rules remain unchanged.
+* <code>BTC<code> as a currency code remains unchanged (1 BTC = 100,000,000 base units)
+* Implementations adopting this standard MUST multiply previously displayed Bitcoin amounts by 100,000,000 to determine the new integer representation.
+
+===Rationale===
+
+'''Usability:'''
+Integer-only displays simplify mental arithmetic and reduce potential confusion or user error.
+
+'''Protocol Alignment:'''
+The Bitcoin protocol inherently counts discrete units. Removing the artificial decimal format aligns user perception with Bitcoin’s actual integral design.
+
+'''Educational Clarity:'''
+Presenting integers ensures newcomers do not mistakenly assume that Bitcoin’s nature is decimal-based. It conveys Bitcoin’s true design from the start.
+
+'''Future-Proofing:'''
+Adopting the smallest unit as the primary measure ensures a consistent standard that can scale smoothly as Bitcoin adoption grows.
+
+'''Perception of Supply:'''
+While the total count of base units is roughly 2.1 quadrillion, this proposal does not alter supply in any way. The change is purely representational. Comparisons can be drawn to other currencies like the Japanese yen or Indonesian rupiah, where high unit counts are standard and not perceived as inflationary.
+
+===Addressing Alternative Approaches===
```

Comment:
I think this paragraph is missing a comparison against the "sats" denomination.

Reply:
This question was brought up another time on the mailing list. I think it is a fair request to ask that it be addressed in this section.